types/types.ts

```diff
@@ -0,0 +1,48 @@
+type OpeningHours = {
```

Comment:
interfaces are preferred since they are more powerful than types, here's a link with more words https://blog.fullstacktraining.com/typescript-interfaces-vs-types/